mod/metrics/set/right/variables.rb

```diff
@@ -36,10 +36,8 @@ def input_metric_name_by_index index
   item_cards.fetch(index, nil).name
 end
 
-
-
 format :html do
-  view :core do |args|
+  view :core, chache: :never do |args|
```

Comment:
typo

Reply:
why wouldn't caching work here?

Reply:
I wanted to make sure that that wasn't the reason that updates on this card had no effect

Reply:
I'm going to guess `chache: :never` didn't fix it :)

Reply:
FYI, view caching is off in development mode by default, so if it's breaking in development (and you haven't explicitly turned it on) it's probably not view caching's fault